unittests.py

```diff
@@ -183,6 +186,32 @@ def testOutputFileFromSourcefile(self):
         self._testFull(['/c', '/P', 'main.cpp'],
                        AnalysisResult.Ok, 'main.cpp', 'main.i')
 
+    def testPreprocessIgnoresOtherArguments(self):
+        # All those inputs must ignore the /Fo, /Fa and /Fm argument according
+        # to the documentation of /E, /P and /EP
+
+        # to file (/P)
+        self._testPreprocessingOutfile(['/c', '/P', 'main.cpp'], 'main.i')
+        self._testPreprocessingOutfile(['/c', '/P', '/FoSome.obj', 'main.cpp'], 'main.i')
+        self._testPreprocessingOutfile(['/c', '/P', '/FaListing.asm', 'main.cpp'], 'main.i')
+        self._testPreprocessingOutfile(['/c', '/P', '/FmMapfile.map', 'main.cpp'], 'main.i')
+
+        # to stdout (/E)
+        self._testPreprocessingOutfile(['/c', '/E', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/E', '/FoSome.obj', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/E', '/FaListing.asm', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/E', '/FmMapfile.map', 'main.cpp'], None)
+        # to stdout (/EP)
+        self._testPreprocessingOutfile(['/c', '/EP', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/EP', '/FoSome.obj', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/EP', '/FaListing.asm', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/EP', '/FmMapfile.map', 'main.cpp'], None)
+        # to stdout (/P /EP)
+        self._testPreprocessingOutfile(['/c', '/P', '/EP', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/P', '/EP', '/FoSome.obj', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/P', '/EP', '/FaListing.asm', 'main.cpp'], None)
+        self._testPreprocessingOutfile(['/c', '/P', '/EP', '/FmMapfile.map', 'main.cpp'], None)
```

Comment:
I'm not sure these last four tests are correct, for me (using Visual Studio 2013), passing both `/P` and `/EP` makes the compiler write the preprocessed output to an `.i` file, too (i.e. `/P` overrules `/EP`). Note that this contradicts the documentation [which says](https://msdn.microsoft.com/en-us/library/becb7sys(v=vs.120).aspx)

> To send the preprocessed output to stdout, with #line directives, use /P and /EP together.